### DISCUSSION TITLE
Can't add user anymore "Database error creating new user"

### DISCUSSION THREAD

**Discussion by CornerAppGuy** [supabase/supabase#26432] [Questions]
Hello, since two days I can't add user anymore to my project. When I try to, I get this error message "Database error creating new user". 
Action that may have caused this : I modified the code from the "User Management Starter" SQL query, adding another user metadata. 
I should have modified the code of the existing "User Management Starter" already running with functions and triggers, but I tried to recreate a whole new one, deleting the already existing functions and triggers. 
Do you have any ideas how this can be fixed ? 
Thank you very much in advance!

**Comment by GaryAustin1** [comment]
Look in the Postgres logs for what your error is.
That usually points you to what needs to be fixed.
Without seeing your function hard to say.

**Reply by CornerAppGuy** [reply]
Hello, nothing in the Postgres log but an error in the Auth log : 
"error": "failed to close prepared statement: ERROR: current transaction is aborted, commands ignored until end of transaction block (SQLSTATE 25P02): ERROR: column \"raw_user_meta_data\" does not exist (SQLSTATE 42703)"
What is strange is that the column "raw_user_meta_data\" does exist in my auth schema .. 

Maybe it's because of a trigger or a function yes. I modified the add user function, to add a metadata input called groupe : 
```plpgsql
begin
  insert into public.profiles (id, full_name, avatar_url, groupe)
  values (new.id, new.raw_user_meta_data->>'full_name', new.raw_user_meta_data->>'avatar_url',raw_user_meta_data->>'groupe');
  return new;
end;
```

**Reply by CornerAppGuy** [reply]
I just saw that I forgot to write "new." before raw_user .... I updated the query and now the problem remains and I get this error message from the Auth log : "ERROR: duplicate key value violates unique constraint \"profiles_pkey\" (SQLSTATE 23505)".
But I checked and all rules are properly set up, I don't understand
